interface/src/app/node/node_search.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * "adage.node_search" module.
+ */
+
+angular.module('adage.node_search', [
+  'ui.router',
+  'ui.bootstrap',
+  'ngResource',
+  'adage.mlmodel.components',
+  'adage.utils'
+])
+
+.config(['$stateProvider', function($stateProvider) {
+  $stateProvider.state('node_search', {
+    url: '/node/search',
```

Comment:
Yeah, I think that is sensible. I think that the code for it wouldn't necessarily need to go in the ``node.js`` file, but if they do not share much in common and it would not be helpful then it makes sense not to do it.